packages/framework/README.md

```diff
@@ -41,15 +41,19 @@ const weeklyComments = workflow(
       cron: CronExpression.EVERY_WEEK,
     }));
 
-    await step.email('weekly-comments', async (controls) => ({
-      subject: `${controls.prefix} - Weekly post comments (${weeklyDigest.events.length})`,
-      body: `Weekly digest: ${weeklyDigest.events.map(({ payload }) => payload.comment).join(', ')}`,
-    }), {
-      // Skip the notification if the user has already seen it
-      skip: () => inAppResponse.seen,
-      // Non-technical stakeholders can modify strongly-validated copy in Novu Cloud
-      inputSchema: z.object({ prefix: z.string().describe('The prefix of the subject.').default('Hi!') }),
-    });
+    await step.email(
+      'weekly-comments',
+      async (controls) => ({
+        subject: `${controls.prefix} - Weekly post comments (${weeklyDigest.events.length})`,
+        body: `Weekly digest: ${weeklyDigest.events.map(({ payload }) => payload.comment).join(', ')}`,
+      }),
+      {
+        // Skip the notification if the user has already seen it
+        skip: () => inAppResponse.seen,
+        // Non-technical stakeholders can modify strongly-validated copy in Novu Cloud
+        inputSchema: z.object({ prefix: z.string().describe('The prefix of the subject.').default('Hi!') }),
+      }
+    );
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Much more readable!‏